items.py

```diff
@@ -184,10 +184,10 @@
 
     # replace public_interface with configured
     if rule.get('input', '') == 'public_interface':
-        rule_copy['input'] = node.metadata['public_interface']
+        rule_copy['input'] = node.metadata['network']['public_interface']
```

Comment:
you cannot assume that this is everywhere. in my setup it is directly on the main system. so either support both (if it is there) or not change this!